paper.md

```diff
@@ -0,0 +1,113 @@
+    corresponding: true # (This is how to denote the corresponding author)
+    affiliation: 2
+  - name: Day Fey
+    affiliation: 2
+  - name: Mary C Norris
+    affiliation: 2
+  - name: Walter Yu
+    affiliation: 3
+  - name: Kristiina Kerge
+    affiliation: 4
+  - name: Kris Haamer
+    affiliation: 4
+  - name: Gina Durante
+    affiliation: 2
+  - name: Brianda Hernandez
+    affiliation: 2
+affiliations:
+ - name: Moore Institute for Plastic Pollution Research, USA
+   index: 1
+ - name: Code for Sacramento and Open Fresno, USA
+   index: 2
+ - name: California Department of Transportation, USA
+   index: 3
+ - name: Let's Do It Foundation, Estonia
+   index: 4
+date: 1 September 2022
+bibliography: paper.bib
+
+---
+
+# Summary
+Although computer vision classification routines have been created for trash, they have not been accessible to most researchers due to the challenges in deploying the models. Trash AI is a web GUI (Graphical User Interface) for serverless computer vision classification of batch images with trash in them hosted at www.trashai.org. With a single batch upload and download, a user can automatically describe the types and quantities of trash in all of their images. 
+
+# Statement of need
+The trash in the environment is a widespread problem that is difficult to measure. Policy makers require high quality data on trash to create effective policies. Classical measurement techniques require surveyors with pen and paper to manually quantify every piece of trash at a site. This method is time-consuming. Scientists are actively trying to address this issue by using imaging to better understand the prevalence and distribution of trash in an `efficient yet effective manner` [@Majchrowska:2022; @Proença:2020; @Moore:2020; @vanLieshout:2020; @WADEAI:2020; @Lynch:2018; @Wuu:2018; @Waterboards:2018]. An app-based reporting of trash using cell phones, laptops, and other devices has been a `valuable solution` [@Lynch:2018]. Applications for AI in detecting trash currently include: images from `bridges` [@vanLieshout:2020], `drone imaging` [@Moore:2020], cameras on `street sweepers` [@Waterboards:2018], and cell phone app based reporting of `trash` [@Lynch:2018]. Although there are many artificial intelligence algorithms developed for trash classification, none are readily accessible to the average litter researcher. The primary limitation is that artificial intelligence (AI) algorithms are primarily run through programming languages (not graphic user interfaces), difficult to deploy without AI expertise, and often live on a server (which costs money to host). New developments in browser-side AI (e.g. tensorflow.js) and serverless architecture (e.g. AWS Lambda) have created the opportunity to have affordable browser-side artificial intelligence in a web GUI alleviating both obstacles. We present Trash AI, an open source service for making computer vision available to anyone with a web browser and images of trash. 
```

Comment:
First sentence:  Delete "The" and begin the sentence with "Trash is..."

Sentence 10--put a comma after each 'e.g.', as follows:  "New developments in browser-side AI (e.g., tensorflow.js) and serverless architecture (e.g., AWS Lambda)"

Sentence 10--Put a comma after 'GUI', as follows:  "...affordable browser-side artificial intelligence in a web GUI, alleviating both obstacles."

Reply:
I want to leave in the "In the environment part because there needs to be a distinction between what we are doing and waste management AI which is a whole different problem that I am not sure this addresses.

Reply:
Made other recommended changes in https://github.com/code4sac/trash-ai/commit/01e6bad6f97ffc431a635a7883476e12524ac3f0